src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -512,6 +509,42 @@ private CompletableFuture<Void> setPoNumberIfMissing(CompositePurchaseOrder comp
     return completedFuture(null);
   }
 
+  public void updatePrefixAndSuffix(CompositePurchaseOrder poFromStorage, CompositePurchaseOrder updatedPo) {
+    String poNumber = "";
+    if(HelperUtils.isPrefixChanged(poFromStorage, updatedPo)) {
+      if(Objects.nonNull(updatedPo.getPoNumberPrefix()) && !updatedPo.getPoNumberPrefix().isEmpty()) {
```

Comment:
I suggest to replace with StringUtils.isNotEmpty(updatedPo.getPoNumberPrefix())

Reply:
fixed